geom/multilinestring.go

```diff
@@ -0,0 +1,43 @@
+package geom
+
+import (
```

Comment:
Can you sort the import as in other packages (or use goimports instead of gofmt).

Reply:
Checked with goimports.